extensions/blocks/map/edit.js

```diff
@@ -230,6 +229,15 @@ class MapEdit extends Component {
 			apiState,
 			apiRequestOutstanding,
 		} = this.state;
+
+		const mapStyle = settings.styles.reduce( ( previewObject, styleObject ) => {
```

Comment:
This construct was a bit tricky for me to understand. Is essentially doing this?

`const mapStyle = settings.styles.find( styleObject => className.includes( styleObect.name ) )`

Reply:
I've done the same thing in multiple different languages this week: using `reduce()` when I should've used `find()`. 🙃 Fixed in 9c35f1d86.